src/build-data/policy/bsi.txt

```diff
@@ -123,6 +127,7 @@ eax
 ocb
 siv
 cfb
+xts
```

Comment:
Should not be prohibited, just not enabled by default.

```suggestion
```

Reply:
I would suggest that we try to stay consistent. For example, why prohibiting the EAX mode, if there is no security flaw (as far as I know). IMO we have three possibilities:
1) We continue prohibiting XTS and all other modes not explicitly recommended
2) We only prohibit modes with discovered security flaws (however,this requires literature research, which is the reason guidelines like the one of BSI exist)
3) We activate the recommended modes but prohibit none.

I think we should get some feedback on these three approaches and apply them to other categories as well. Until then I think its the best approach to stay consistent with approach 1), i.e., prohibiting `xts`. Or is `xts` some required dependency?

Reply:
> I would suggest that we try to stay consistent. For example, why prohibiting the EAX mode, if there is no security flaw (as far as I know). IMO we have three possibilities:
> 
> 1) We continue prohibiting XTS and all other modes not explicitly recommended

I don't like this option.

> 2) We only prohibit modes with discovered security flaws (however,this requires literature research, which is the reason guidelines like the one of the BSI exist)

I could **easily** live with this choice.

> 3) We activate the recommended modes but prohibit none.

This seems the best and the soundest approach. Though might want to combine with "prohibit nodes work discovered [practical, exploitable] security flaws".



> I think we should get some feedback on these three approaches and apply them to other categories as well.

Indeed.